### DISCUSSION TITLE
The "title" meta tag does not seem to work in RC 13

### DISCUSSION THREAD

**Discussion by murugappanrm** [nuxt/nuxt#16437] [Questions] [answered]
I tried defining the meta tag "title" but it does not seem to work in RC13. I defined the meta "title" in the nuxt.config.ts file as follows:

```
export default defineNuxtConfig({
  css: ["assets/css/main.css"],
  app: {
    head: {
      title: "Resume App",
      meta: [
        { charset: "utf8" },
        {
          name: "viewport",
          content: "width=device-width, initial-scale=1, shrink-to-fit:'no'",
        },
        {
          name: "author",
          content: "Murugappan",
        },
        {
          name: "description",
          content: "Resume Apps",
        },
      ],
    },
  },
});

```
Also tried defining it in the layout default.vue as follows: 

```
<template>
  <div>
    <slot />
  </div>
</template>

<script setup>
  useHead({
    title: "Muru's Resume",
  });
</script>

<style scoped></style>
```

Please help.

**Comment by TheAlexLichter** [comment, marked as answer]
Hey! Could you kindly provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

Answer in the comments below.

**Reply by murugappanrm** [reply]
@manniL 

I am a newbie and I tried on stackblitz and this is the URL:

https://stackblitz.com/edit/github-molejd-u1rv3x?file=layouts/default.vue

**Reply by TheAlexLichter** [reply]
No problem!

* You use just a single layout. Instead, leverage your `app.vue` (if you have just one layout). If you want to have multiple, adapt your `app.vue` so it actually renders the layout:

```vue
<template>
  <NuxtLayout>
    <NuxtPage />  <!-- If you have pages/, else just your app content -->
  </NuxtLayout>
</template>

```

* The `NuxtWelcome` component is overriding the title. Just remove it.

**Reply by murugappanrm** [reply]
Fantastic @manniL  it works now. Thank you so much.

**Reply by TheAlexLichter** [reply]
You are welcome ☺️

**Reply by murugappanrm** [reply]
Sorry @manniL 

I just encountered another problem. I defined my css and script files under "link" and "script" objects in useHead() but both are not detected and used by the app. I tried on stackblitz and this is the URL:

https://stackblitz.com/edit/github-go1ntq-w5tp7g?file=layouts%2Fdefault.vue,nuxt.config.ts

**Reply by TheAlexLichter** [reply]
Define them in your `nuxt.config.ts`. See e.g. https://v3.nuxtjs.org/api/configuration/nuxt-config#css

**Reply by murugappanrm** [reply]
But this is for global. What is i need set different styles or scripts for different layouts?

**Reply by TheAlexLichter** [reply]
For styles, you can e.g. use `scoped css` of your components.

**Reply by murugappanrm** [reply]
Ah I forgot about  the scoped styles. I guess for scripts it cannot be done?

**Reply by TheAlexLichter** [reply]
No but you can load these via `useHead` though.

**Comment by murugappanrm** [comment]
@manniL  This is the URL for your use:

https://stackblitz.com/edit/github-molejd-u1rv3x?file=layouts/default.vue

**Comment by murugappanrm** [comment]
@manniL 

I am a newbie and dont much about using these stackblitz and codesandbox stuff. Anyway i tried on stackblitz and this is the URL:

https://stackblitz.com/edit/github-molejd-u1rv3x?file=layouts/default.vue
